Makefile

```diff
@@ -130,6 +131,14 @@ requirements.txt: pyproject.toml
 	echo "" >> requirements.txt
 	cp requirements.txt dist/package-$(PACKAGE_VERSION)-requirements.txt
 
+# Audit the currently installed packages.
+.PHONY: audit
+audit:
+	if ! $$(python -c "import pip_audit" &> /dev/null); then \
+	  echo "No package pip_audit installed, upgrade your environment!" && exit 1; \
+	fi;
+	python -m pip_audit --skip-editable --desc on --fix --dry-run
```

Comment:
Can you please add comments for these cmd line options?

Reply:
Commit https://github.com/jenstroeger/python-package-template/pull/307/commits/12b27833f1328491d2e03023c0281bf667ca8864.